src/Auth0.ManagementApi/Models/ClientResourceServerAssociation.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Auth0.ManagementApi.Models
+{
+    public class ClientResourceServerAssociation
+    {
+        /// <summary>
+        /// The resource server identifier.
```

Comment:
I would add ` (API)` after `The resource server`, to match the dashboard and docs more closely.